poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.8.4 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.0.1 and should not be changed by hand.
```

Comment:
Non-blocker: Make sure our poetry version matches what's in the Dockerfile which is currently `1.8.5`

Reply:
Don't want to block here, so not requesting any changes.

Reply:
Our Dockerfile is on `1.8.5`?